crmprtd/__init__.py

```diff
@@ -51,3 +55,26 @@ def f_retry(*args, **kwargs):
             return f(*args, **kwargs)
         return f_retry  # true decorator
     return deco_retry
+
+
+def setup_logging(log_conf, log, error_email, log_level, name):
+    if log_conf:
+        with open(log_conf, 'rb') as f:
+            log_c = yaml.load(f)
```

Comment:
Let's change the name of `log_c` to be `base_config`. So we start with a base config and then optionally modify it.